task-2-SERVLETS/Self-check.md

```diff
@@ -0,0 +1,13 @@
+1. What is the Servlet Container? List some of them.
+2. What is the life-cycle of a servlet?
+3. What are life-cycle methods for a servlet do you know and what is their functions?
+4. What is REST?
+5. What HTTP methods are used according to the REST?
+6. Describe what http-statuses are used in the responses and what server behaviour they mean according to the REST.
```

Comment:
Suppose this would be better:
6. Describe which http-statuses are used in responses and what kind of server behavior do they mean according to the REST.

Reply:
```suggestion
6. Describe which http-statuses are used in responses and what kind of server behavior do they mean according to the REST.
```